src/theme/buttons.ts

```diff
@@ -30,11 +30,19 @@ const buttons: Buttons = {
       color: colors.blueberryClassic,
       hoverColor: colors.radiantBlueberry,
       activeColor: colors.dietBlueberry,
+      decoration: 'none',
     },
     danger: {
       color: colors.cherry,
       hoverColor: colors.strawberry,
       activeColor: colors.pumpkin,
+      decoration: 'none',
+    },
+    footnote: {
```

Comment:
I would use `secondary` instead of `footnote` because it's a more generic name and we were used to having secondary in this grey-ish colours

Reply:
I'd rather use semantic naming, 'secondary' doesn't really say much 🤷. But unless there's other opinions, i suppose you're the boss 😄

Reply:
well, 'footnote' has some semantic meaning, but it doesn't have to be true for other applications and also what happens when you'll need to use this style in the header or somewhere else than footnote :D

Reply:
Yeah, that's actually very fair point. And I see that 'secondary' is also used for the text element of the same color (not footnote), so 'secondary' it is 👍